templates/resources/apigateway_method.md.tmpl

```diff
@@ -0,0 +1,31 @@
+---
+page_title: "{{.Name}} {{.Type}} - {{.ProviderName}}"
+subcategory: ""
+description: |-
+{{ .Description | plainmarkdown | trimspace | prefixlines "  " }}
+---
+
+# {{.Name}} ({{.Type}})
+
+{{ .Description | trimspace }}
+
+## Example Usage
+
+### First example
+Description about the first example
+{{ tffile (printf "examples/resources/%s/apigateway_method_basic_example.tf" .Name)}}
+
+### Second example
```

Comment:
same here

Reply:
Hey! Thank you for the review, just added the changes you mentioned